apps/pipeline/test/unit/pipeline/writer/table_writer/helper/prestige_helper_test.exs

```diff
@@ -0,0 +1,54 @@
+defmodule Pipeline.Writer.TableWriter.Helper.PrestigeHelperTest do
+  use ExUnit.Case
+  use Placebo
+
+  alias Pipeline.Writer.TableWriter.Helper.PrestigeHelper
+
+  @expected_table_data %{
+    "column_1" => "some column_1 data",
+    "column_2" => "some column_2 data"
+  }
+
+  @tag capture_log: true
+  test "should return the data by executing the when the query statement is passed" do
+    allow(Prestige.new_session(any()), return: :connection)
+    allow(Prestige.execute(any(), any()), return: @expected_table_data)
```

Comment:
Seems like you make these allowances more strict: 
`allow(Prestige.execute(:connection, "whatever"), return: @expected_table_data)`

This way you know that the proper args are getting passed.